std/haxe/math/bigint/MutableBigInt.hx

```diff
@@ -0,0 +1,426 @@
+		var a:MutableBigInt_ = this;
+		a.copyFrom(other);
+	}
+
+	public inline function getBit(index:Int):Int {
+		return BigIntArithmetic.getBit(this, index);
+	}
+
+	public static function fromBigEndianBytesUnsigned(value:Bytes):MutableBigInt {
+		var r = new MutableBigInt_();
+		r.setFromBigEndianBytesUnsigned(value);
+		return new MutableBigInt(r);
+	}
+
+	public static function fromLittleEndianBytesUnsigned(value:Bytes):MutableBigInt {
+		var r = new MutableBigInt_();
+		r.setFromLittleEndianBytesUnsigned(value);
+		return new MutableBigInt(r);
+	}
+
+	public function abs():MutableBigInt {
+		return this.abs();
+	}
+
+	public function gcd(b:BigInt):MutableBigInt {
+		return this.gcd(b);
+	}
+
+	public function pow(exponent:UInt):MutableBigInt {
+		return this.pow(exponent);
+	}
+
+	public function modPow(exponent:BigInt, modulus:BigInt):MutableBigInt {
+		return this.modPow(exponent, modulus);
+	}
+
+	public function isProbablePrime(tolerance:UInt):Bool {
+		return this.isProbablePrime(tolerance);
+	}
+
+	public function getLowestSetBit():Int {
+		return this.getLowestSetBit();
+	}
+
+	public function bitLength():Int {
+		return this.bitLength();
+	}
+
+	//-----------------------------------------------------------------------
+	// Operators
+	//-----------------------------------------------------------------------
+	// The declaration order of the operations is significant in Haxe.
+	// Recommended order is:
+	//	* MutableBigInt <binOp=> Int
+	//	* MutableBigInt <binOp=> BigInt
+	//	* MutableBigInt <binOp=> MutableBigInt
+	//	* MutableBigInt <binOp> Int
+	//	* MutableBigInt <binOp> BigInt
+	//	* MutableBigInt <binOp> MutableBigInt
+	// Unary negation
+	@:op(-A) @:noCompletion public static inline function negate_(a:MutableBigInt):BigInt {
+		return new BigInt(BigInt_.negate1(a));
+	}
+
+	// Binary equality
+	@:op(A == B) @:noCompletion public static inline function eqInt_(a:MutableBigInt, b:Int):Bool {
+		return BigInt_.equals2Int(a, b);
+	}
+
+	@:op(A == B) @:noCompletion public static inline function eq_(a:MutableBigInt, b:BigInt):Bool {
+		return BigInt_.equals2(a, b);
+	}
+
+	@:op(A == B) @:noCompletion public static inline function eqMutable_(a:MutableBigInt, b:MutableBigInt):Bool {
+		return BigInt_.equals2(a, b);
+	}
+
+	// Binary inequality
+	@:op(A != B) @:noCompletion public static inline function ineqInt_(a:MutableBigInt, b:Int):Bool {
+		return !BigInt_.equals2Int(a, b);
+	}
+
+	@:op(A != B) @:noCompletion public static inline function ineq_(a:MutableBigInt, b:BigInt):Bool {
+		return !BigInt_.equals2(a, b);
+	}
+
+	@:op(A != B) @:noCompletion public static inline function ineqMutable_(a:MutableBigInt, b:MutableBigInt):Bool {
+		return !BigInt_.equals2(a, b);
+	}
+
+	// Binary less than
+	@:op(A < B) @:noCompletion public static inline function ltInt_(a:MutableBigInt, b:Int):Bool {
+		return BigIntArithmetic.compareInt(a, b) < 0;
+	}
+
+	@:op(A < B) @:noCompletion public static inline function lt_(a:MutableBigInt, b:BigInt):Bool {
```

Comment:
Might be good to also add @:noDoc to these functions

Reply:
Thanks, I added @:noDoc